### PR TITLE
Tron network

### PR DESCRIPTION
**Pull request by cryptoiner**
- [x] Tron network added
- [x] Shasta network added

### CODE REVIEW

File: src/chainIds.ts [hunk 1]
```diff
@@ -9,4 +9,6 @@ export enum ChainId {
   RSKTestnet = 31,
   Polygon = 137,
   Mumbai = 80001,
+  Tron = 7000,
```
Comment (djaciel) [inline src/chainIds.ts:12]: Is this ok? I suppose that any other network uses this chainId

File: src/networks/index.ts [hunk 2]
```diff
@@ -123,6 +123,30 @@ export const networksByChainId: INetworkByChainId = {
...
+    txExplorerUrl: 'https://shasta.tronscan.org/#/transaction/',
+    isMainnet: false,
+    enabled: true,
+  },
+  [ChainId.Tron]: {
+    name: 'Tron',
+    chainId: ChainId.Tron,
+    payrollAddress: '',
+    routerAddress: '',
+    wCurrencyAddress: '',
```
Comment (djaciel) [inline src/networks/index.ts:143]: this is the wtrx contract `TNUC9Qb1rRpS5CbWLmNMxXBjyFoydXjWFR`

File: src/tokens/shasta.ts [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+export const tokens: IToken[] = [
+  {
+    name: 'TRX',
+    address: '0x0000000000000000000000000000000000000000',
+    chainId: ChainId.Shasta,
+    enabled: true,
+    decimals: 6,
+    symbol: 'TRX',
+  },
```
Comment (djaciel) [inline src/tokens/shasta.ts:12]: add pls this token
`{
    name: 'USDT',
    address: 'TSsjPxRVQxN5CKk59Gfxyme5vhgp3ymSKq',
    chainId: ChainId.Shasta,
    enabled: true,
    decimals: 18,
    symbol: 'USDT',
  }`

File: src/tokens/shasta.ts [hunk 4]
```diff
@@ -0,0 +1,13 @@
+import { ChainId } from '../chainIds';
+import { IToken } from '../interfaces';
+
+export const tokens: IToken[] = [
+  {
+    name: 'TRX',
+    address: '0x0000000000000000000000000000000000000000',
```
Comment (djaciel) [inline src/tokens/shasta.ts:7]: this is the zero address for tron `T9yD14Nj9j7xAB4dbGeiX9h8unkKHxuWwb`

File: src/tokens/tron.ts [hunk 5]
```diff
@@ -0,0 +1,13 @@
+import { ChainId } from '../chainIds';
+import { IToken } from '../interfaces';
+
+export const tokens: IToken[] = [
+  {
+    name: 'TRX',
+    address: '0x0000000000000000000000000000000000000000',
```
Comment (djaciel) [inline src/tokens/tron.ts:7]: change zero address `T9yD14Nj9j7xAB4dbGeiX9h8unkKHxuWwb`

ask to @Ignacio-87 what tokens we'll need to add in mainnet

**Review by djaciel — CHANGES_REQUESTED** [review]
LGTM, I just left a couple of changes

File: src/networks/index.ts (hunk 2 above)
Comment (cryptoiner) [reply src/networks/index.ts]: Done!

File: src/networks/index.ts [hunk 6]
```diff
@@ -123,6 +123,30 @@ export const networksByChainId: INetworkByChainId = {
...
+    addressExplorerUrl: 'https://shasta.tronscan.org/#/address/',
+    txExplorerUrl: 'https://shasta.tronscan.org/#/transaction/',
+    isMainnet: false,
+    enabled: true,
+  },
+  [ChainId.Tron]: {
+    name: 'Tron',
+    chainId: ChainId.Tron,
+    payrollAddress: '',
+    routerAddress: '',
```
[the remark this answers is not in the document]
Comment (cryptoiner) [reply src/networks/index.ts]: Done!

File: src/tokens/shasta.ts (hunk 3 above)
Comment (cryptoiner) [reply src/tokens/shasta.ts:12]: Done!

File: src/tokens/shasta.ts (hunk 4 above)
Comment (cryptoiner) [reply src/tokens/shasta.ts]: Done!

File: src/tokens/tron.ts (hunk 5 above)
Comment (cryptoiner) [reply src/tokens/tron.ts]: Done!

**Review by djaciel — APPROVED** [review]
nice 💯

**Pull request merged**